src/Microsoft.TestPlatform.TestHostProvider/Hosting/TestHostManagerCallbacks.cs

```diff
@@ -14,6 +15,8 @@ namespace Microsoft.TestPlatform.TestHostProvider.Hosting;
 
 internal class TestHostManagerCallbacks
 {
+    const int E_HANDLE = unchecked((int)0x80070006);
```

Comment:
Nit: explicit modifier?

```suggestion
    private const int E_HANDLE = unchecked((int)0x80070006);
```